server/graphql/common/expenses.ts

```diff
@@ -396,6 +396,14 @@ export const canSeeExpenseSecurityChecks: ExpensePermissionEvaluator = async (re
   return remoteUserMeetsOneCondition(req, expense, [isHostAdmin]);
 };
 
+export const canSeeDraftKey: ExpensePermissionEvaluator = async (req, expense) => {
+  if (expense.status !== expenseStatus.DRAFT) {
+    return false;
+  }
+
+  return canSeeExpenseSecurityChecks(req, expense);
```

Comment:
I see a risk in defining such a dependency, as developers iterating on `canSeeExpenseSecurityChecks` may not be aware of it. I think this is a case for "explicit" is better than "re-usable", a copy-paste wouldn't hurt.

Plus, security checks are limited to trusted hosts, I don't see why we would prevent this one for other hosts.